composer/utils/file_helpers.py

```diff
@@ -349,6 +349,8 @@ def maybe_create_object_store_from_uri(uri: str) -> Optional[ObjectStore]:
         return GCSObjectStore(bucket=bucket_name)
     elif backend == 'oci':
         return OCIObjectStore(bucket=bucket_name)
+    elif backend == 'dbfs' and path.startswith('Volumes/'):
+        return UCObjectStore(uri=uri)
     else:
         raise NotImplementedError(f'There is no implementation for the cloud backend {backend} via URI. Please use '
                                   's3 or one of the supported object stores')
```

Comment:
Not part of this PR, but we should change the exception message here. We can omit keyword `s3` since we support multiple cloud backends.